xtable-core/pom.xml

```diff
@@ -138,12 +138,23 @@
             <scope>provided</scope>
         </dependency>
 
+        <!-- AWS Glue dependencies -->
```

Comment:
I think it would be best to create a new module for the Glue related code. That will allow our users to only pull in the dependencies that they require.